pages/common/routers/form.js

```diff
@@ -98,7 +98,7 @@ module.exports = ({
       format = format || identity;
       all[key] = format(value);
       return all;
-    }, {});
+    }, {}));
```

Comment:
I think I'd prefer this code if it was formatted as a number of explicit steps rather than nested function calls that are performing multiple discrete functions in a single statement. It's getting highly non-obvious.

```js

req.form.values = pick(req.body, Object.keys(req.form.schema));
req.form.values = mapValues(req.form.values, (value, key) => {
  const nullValue = req.form.schema[key].nullValue;
  return value || nullValue;
});
req.form.values = mapValues(req.form.values, (value, key) => {
  const format = req.form.schema[key].format || identity;
  return format(value);
});
req.form.values = mapValues(req.form.values, trim);
req.form.values = cleanModel(req.form.values);
```

Reply:
You'd probably want to pull those out into named functions to make it even more obvious, but it took me 10 minutes of unpicking the code to work out what actual steps were being performed in there.

Reply:
At the _very_ least, add the `cleanModel` as another line.

```js
req.form.values = reduce(...everything as it was before...);
req.form.values = cleanModel(req.form.values);
```